### PR TITLE
add maxAngle

### PR DESCRIPTION
**Pull request by Stormiix**
#46 
For some reason, the maxAngle was removed. This pull request should fix that.

### CODE REVIEW

**Comment by Stormiix** [comment]
This definetly needs some testing before merging. Can you please handle that @Hameck ?

File: src/ping360_sonar/node.py [hunk 1]
```diff
@@ -181,7 +184,7 @@ def main():
                     else:
                         pointColor = 0
                     for k in np.linspace(0, step, 8 * step):
-                        theta = 2 * pi * (angle + k) / 400.0
+                        theta = 2 * pi * (angle + k) / float(400)
```
Comment (Stormiix) [inline src/ping360_sonar/node.py]: vaut mieux mettre 400.0, non ?

Comment (Hameck) [reply src/ping360_sonar/node.py]: oui!

**Comment by Stormiix** [comment]
This closes #46 .

**Pull request merged**